otterdog/eclipse-edc.jsonnet

```diff
@@ -458,6 +458,29 @@ orgs.newOrg('eclipse-edc') {
         },
       ],
     },
+    orgs.newRepo('Technology-HuaweiCloud') {
+      allow_rebase_merge: false,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      has_discussions: true,
+      has_wiki: false,
+      squash_merge_commit_title: "PR_TITLE",
+      web_commit_signoff_required: false,
+      workflows+: {
+        default_workflow_permissions: "write",
+      },
+      secrets: [
+        orgs.newRepoSecret('DISCORD_WEBHOOK_GITHUB') {
```

Comment:
are these secrets required? if so, we need to resolve them with references to our credential store. The JENKINS token is easy to setup but the discord webhook will need to be setup by the project team and then shared with us via a secure channel.

We could also start without secrets and add them as needed.

Reply:
We don't need the discord webhook anymore...